### PR TITLE
Improve composer require on readme

### PR DESCRIPTION
**Pull request by Soullivaneuh**

### CODE REVIEW

**Comment by vinkla** [comment]
Much better :+1:

File: README.md [hunk 1]
```diff
@@ -10,12 +10,8 @@ Simple object oriented wrapper for Packagist API.
...
 The recommended way to install Packagist API is through composer:
 
-```json
-{
-    "require": {
-        "knplabs/packagist-api": "1.*@dev"
-    }
-}
+```bash
+composer require knplabs/packagist-api
```
Comment (umpirsky) [inline README.md]: Great, just maybe better to put:

```bash
$ composer require knplabs/packagist-api
```

It's more clear that it is a terminal.

Comment (Soullivaneuh) [inline README.md]: Can you this, but in many projects the `$` is avoided.

**Comment by Soullivaneuh** [comment]
@umpirsky done.

**Comment by umpirsky** [comment]
Merged, thanks @Soullivaneuh! :+1:

**Pull request merged**